data/json/items/ammo.json

```diff
@@ -179,6 +179,7 @@
         "weight" : 80,
         "ammo_type" : "plutonium",
         "count" : 5,
+        "energy": 200,
```

Comment:
Is that just 4 times as much as gasoline? Is it per volume or unit? If per unit, then gasoline is >50 times more efficient.